tasks/clone.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Clone {{ repo.url }}
+  include_role:
+    name: adamyala.ansible-role-clone-repo
```

Comment:
Is there a benefit we get when we use this role instead of the default Ansible [Git module](https://docs.ansible.com/ansible/latest/modules/git_module.html)?

Reply:
It provides a way to have the git stuff done in a more packaged and re-usable way.  Very similar to what [onaio.ansible-git-clone](https://github.com/onaio/ansible-git-clone) is for.  I am suing this instead of `onaio.ansible-git-clone` because right now onaio.ansible-git-clone is broken for ssh agent forwarding.